AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginFunctionalTests/GraphQLConnectionScenario6Tests.swift

```diff
@@ -59,50 +59,39 @@ class GraphQLConnectionScenario6Tests: XCTestCase {
         await Amplify.reset()
     }
 
-    func testGetBlogThenFetchPostsThenFetchComments() {
-        guard let blog = createBlog(name: "name"),
-              let post1 = createPost(title: "title", blog: blog),
-              let post2 = createPost(title: "title", blog: blog),
-              let comment1post1 = createComment(post: post1, content: "content"),
-              let comment2post1 = createComment(post: post1, content: "content") else {
+    func testGetBlogThenFetchPostsThenFetchComments() async throws {
+        guard let blog = try await createBlog(name: "name"),
+              let post1 = try await createPost(title: "title", blog: blog),
+              let post2 = try await createPost(title: "title", blog: blog),
```

Comment:
Seeing warning : `Immutable value 'post2' was never used; consider replacing with '_' or removing it`